x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -139,25 +161,47 @@ export const requiredFieldsForActions = [
 ];
 
 // TODO: Once we are past experimental phase this code should be removed
```

Comment:
Is this TODO still applicable? What is the experimental phase? Is that RAC stuff? If so, no worries, it's fine as is.

Reply:
That's a great question, idk if that phase has changed or not, it's tied into the rule registry stuff. @spong @yctercero is this being modified at some point in the future still?

Reply:
This should be removed here real soon (~1 week?) as we move over to use the Rule Registry.